chatbot.py

```diff
@@ -128,27 +128,25 @@
         exit()
     elif translate in cmd5:
         print("here")
-        owm = pyowm.OWM(config.weather_api_key)
-        observation = owm.weather_at_place('Bangalore, IN')
-        observation_list = owm.weather_around_coords(12.972442, 77.580643)
-        w = observation.get_weather()
-        w.get_wind()
-        w.get_humidity()
-        w.get_temperature('celsius')
-        print(w)
-        print(w.get_wind())
-        print(w.get_humidity())
-        print(w.get_temperature('celsius'))
-        engine.say(w.get_wind())
-        engine.runAndWait()
-        engine.say('humidity')
-        engine.runAndWait()
-        engine.say(w.get_humidity())
-        engine.runAndWait()
-        engine.say('temperature')
-        engine.runAndWait()
-        engine.say(w.get_temperature('celsius'))
+        url="http://api.openweathermap.org/data/2.5/weather?lat={}&lon={}&appid={}".format(latitude,longitude,config.weather_api_key)
+        r = requests.get(url)
+        x = r.json()
+        city = x['name']
+        windSpeed = x['wind']['speed']
+        skyDescription = x['weather'][0]['description']
+        maxTemperature = x['main']['temp_max']
+        minTemperature = x['main']['temp_min']
+        temp = x['main']['temp']
+        humidity = x['main']['humidity']
+        pressure = x['main']['pressure']
+        tempCel = round(temp-273.15,2)
```

Comment:
try using units parameter listed in api documentation https://openweathermap.org/current. Also it will be nice we add temp unit parameter in config file with default as celsius.